src/sections/home-text-with-images.liquid

```diff
@@ -55,8 +55,11 @@
     color: #fff;
   }
   {% for block in section.blocks %}
-    .text-columns-with-images .block-{{ block.id }} .image-container::after {
-      opacity: {{ block.settings.image_overlay_opacity | times: 0.1 }};
+    .block-{{ block.id }} .image-container::after {
+      opacity: {{ section.settings.image_overlay_opacity | times: 0.1 }};
+    }
```

Comment:
@rdnvol actually you don't need this rule to be posted for every block, so you may move it higher (exclude from blocks iteration)